src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger/config.jelly

```diff
@@ -242,7 +252,7 @@
                             checkUrl="'descriptorByName/com.sonyericsson.hudson.plugins.gerrit.trigger.hudsontrigger.GerritTrigger/urlCheck?value='+escape(this.value)"/>
                     </f:entry>
                 </f:optionalBlock>
-            </table>
+            </p:blockWrapperTableWidth>
         </f:entry>
 
```

Comment:
The actual main configuration form is below this and contains lots of table layouts. What about that?